Formula/libgpg-error.rb

```diff
@@ -18,7 +19,24 @@ class LibgpgError < Formula
     sha256 x86_64_linux:  "b862b10316313dabc9ddc7774e37c41c0700b886d361b86a1485052287b73fd0"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gettext" => :build
```

Comment:
Can we try building without this? In my experience this isn't actually needed except when building locally on systems that happen to have it installed.

Reply:
I have made the patch directly to configure instead. This way we don't need to add any additional build dependencies.